e2etest/rewarding_test.go

```diff
@@ -572,26 +589,23 @@ func updateExpectationWithPendingClaimList(
 }
 
 func newConfig(
-	chainDBPath,
-	trieDBPath,
-	indexDBPath string,
 	producerPriKey crypto.PrivateKey,
 	networkPort,
 	apiPort int,
-	numNodes uint64,
+	HTTPAdminPort int,
 ) config.Config {
 	cfg := config.Default
 
+	cfg.Plugins[config.GatewayPlugin] = true
```

Comment:
if enable plugin, need to delete it in defer{}, otherwise it might cause problem for later tests.
search "// clear the gateway" you will see